dashboard/test/factories/course_versions.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: course_versions
+#
+#  id                 :integer          not null, primary key
+#  version_name       :string(255)
+#  properties         :text(65535)
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  content_root_type  :string(255)
+#  content_root_id    :integer
+#  course_offering_id :integer
+#
+# Indexes
+#
+#  index_course_versions_on_content_root_type_and_content_root_id  (content_root_type,content_root_id)
+#  index_course_versions_on_course_offering_id                     (course_offering_id)
+#  index_course_versions_on_course_offering_id_and_version_name    (course_offering_id,version_name) UNIQUE
+#
+
+FactoryGirl.define do
+  factory :course_version do
```

Comment:
Should every course version have a course? if so, `:course_version` should have a `course_offering` field here, kind of like `:lesson` has for `script`: https://github.com/code-dot-org/code-dot-org/blob/a7c45d96b1e959a92560169e071fdc3c5055664f/dashboard/test/factories/factories.rb#L755-L757